### DISCUSSION TITLE
Hide `up_to_date` versions entirely?

### DISCUSSION THREAD

**Discussion by alxndr** [vuki656/package-info.nvim#44] [Ideas]
Love the plugin!!

Would it be feasible to add an option whereby only the `outdated` versions are shown, and there is no  virtual text on `up_to_date` lines?

**Comment by vuki656** [comment]
Hey, that is a good idea.

I created an issue and added it to the roadmap. 

Will definitely implement this soon. 

https://github.com/vuki656/package-info.nvim/issues/45

**Reply by alxndr** [reply]
Thanks for implementing!
